### PR TITLE
Minor documentation issues + gh-actions workflow

### PR DESCRIPTION
**Pull request by CBroz1**
Adds documentation to address
- #8 
- #9 
- #10 
- #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,10 +13,11 @@
...
 In both of the following diagrams, the trial table starts immediately downstream from
-***Session***. In one case, Sessions are first segmented into trials, and then 
-segmented into events. This might be appropriate, for example, in a paradigm with 
-repeated conditions and response behaviors associated with different conditions. In the 
-next, Sessions are directly upstream from both Trials and Events. This might be appropriate for a paradigm that recorded events within naturalistic free behavior. We  provide an
+***Session***. In one case, Sessions are first segmented into trials, and then segmented
+into events. This might be appropriate, for example, in a paradigm with repeated
+conditions and response behaviors associated with different conditions. In the next,
+Sessions are directly upstream from both Trials and Events. This might be appropriate
+for a paradigm that recorded events within naturalistic free behavior. We provide an
```
Comment (kabilar) [inline README.md:20]: Please rephrase as there is only 1 diagram.

Comment (CBroz1) [reply README.md]: All set

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1!  Added bullet points to the docstring.  Also, please update version and changelog.

**Comment by CBroz1** [comment]
> Also, please update version and changelog

All set. Thanks for your review!

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1!

**Pull request merged**